modules/standard/Reflection.chpl

```diff
@@ -60,7 +60,7 @@ proc numFields(type t) param : int
 proc getFieldName(type t, param idx:int) param : string
   return __primitive("field num to name", checkQueryT(t), idx+1);
 
-/* Get the name of the ith field in a class or record.
+/* Get the name of the field at `i` in a class or record.
```

Comment:
I don't think this one needed an update, but it's okay to leave the change you've made